### PR TITLE
py-segmentation-models-pytorch: add v0.3.2

### PR DESCRIPTION
**Pull request by adamjstewart**
https://github.com/qubvel/segmentation_models.pytorch/releases/tag/v0.3.2

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-timm/package.py [hunk 1]
```diff
@@ -12,13 +12,17 @@ class PyTimm(PythonPackage):
     homepage = "https://github.com/rwightman/pytorch-image-models"
     pypi = "timm/timm-0.4.12.tar.gz"
 
+    version("0.6.12", sha256="8f1747121598e06a1ea2d00df16d332cc284cdd4596bdc136b490a2122d3aa91")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-timm/package.py:15]: Confirmed this sha256.

File: var/spack/repos/builtin/packages/py-segmentation-models-pytorch/package.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ class PySegmentationModelsPytorch(PythonPackage):
     homepage = "https://github.com/qubvel/segmentation_models.pytorch"
     pypi = "segmentation_models_pytorch/segmentation_models_pytorch-0.2.0.tar.gz"
 
+    version("0.3.2", sha256="8372733e57a10cb8f6b9e18a20577fbb3fb83549b6945664dc774a9b6d3ecd13")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-segmentation-models-pytorch/package.py:15]: Confirmed this sha256.

**Review by tldahlgren — DISMISSED** [review]

**Comment by adamjstewart** [comment]
@spackbot run pipeline

**Comment by adamjstewart** [comment]
@spackbot run pipeline

**Review by alalazo — APPROVED** [review]

**Pull request merged**